src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
+    private static final Random randomInt = new Random();
+    private static final int MAX_RANDOM_NUMBER = 100;
+    private final int lotteryDraw;
```

Comment:
I dont think we need this variable here, lets remove it, it is not describing lottery state

Reply:
Are you talking about MAX_RANDOM_NUMBER and the lottery? So. They are not needed at all in this committee. They were necessary fields in the commit # 52624087605a7f8ec507573922e9779d05152300. I then had more designers who allowed me to use the lottery dynamically.